app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,22 @@
+<style>
```

Comment:
This is a stop-gap design. Once we hear from Design, we shall update accordingly.